hangman.rb

```diff
@@ -0,0 +1,124 @@
+class Hangman
+
+  def initialize
+    @now_playing = true
+    @count = 0
+
+    # Board Display
+    @row1 = "|     _________"
+    @row2 = "|     |/      |"
+    @row3 = "|     |      "
+    @row4 = "|     |      "
+    @row5 = "|     |       "
+    @row6 = "|     |      "
+    @row7 = "|     |"
+    @row8 = "| ____|_____"
+
+    @board = [@row1, @row2, @row3, @row4, @row5, @row6, @row7, @row8]
+
+    @dictionary = ["hotdog", "elephant", "computer", "developer", "baby", "dragon", "classmate", "program", "teamwork", "fun"]
+
+    start
+  end
+
+  def start
+    puts "Let's play a game of Hangman!"
+    puts @board
+
+    pick_word
+    @word = ["_ "] * @answer_letters.length
+    puts @word.join
+    play
+  end
+
+  def play
+    while @now_playing == true
+      get_guess
+      check_guess
+      puts @board
+      puts @word.join
+      win
+    end
+  end
+
+  def pick_word
+    #randomly selects word
+    answer = @dictionary[rand(0...@dictionary.length)]
+    #convert individual letters of word to array
+    @answer_letters = answer.split(//)
+
+    puts "WORD IS:" + " #{@answer_letters.join}"
+  end
+
+#___________________________________
+# GUESS CODE
+
+  def get_guess
+    puts "Guess a letter! \nType quit to exit the game."
+    input = gets.chomp.downcase
+
+    if input == "quit"
+      puts "Bye!"
+      exit
+    end
+    @guess_letters = input.split(//)
```

Comment:
Are you expecting your users to guess more than one letter at a time? 

Otherwise, I'm confused as to why you are using the split method here. Chomping the input should be enough to get rid of any /n or /r. If not you might consider using the chop method and seeing if it will suit your needs: http://ruby-doc.org/core-2.2.0/String.html#method-i-chop

If you do want to process more than one guess at a time you might want to chance the language to your players to reflect that.